tests/cluster/sql/test_deliverysql.py

```diff
@@ -309,6 +309,19 @@ def test19_upsert_to_table_overwrite_empty_target(self):
             expected_data=expectedData,
         )
 
+    def test20_big_data_set(self):
```

Comment:
This test has not directly anything to do with the table lock option? It was just missing right?

Reply:
Correct. It was just a missing test!